src/main/main.c

```diff
@@ -29,8 +29,14 @@
 
 void run(void);
 
-int main(void)
+int main(int argc, char * argv[])
 {
+#ifdef SIMULATOR_BUILD
+    targetParseArgs(argc, argv);
+#else
+  UNUSED(argc);
+  UNUSED(argv);
```

Comment:
Found another  one, sorry 😀
```suggestion
    UNUSED(argc);
    UNUSED(argv);
```